tools/pd-ctl/pdctl/command/keyspace_command.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+const keyspacePrefix = "pd/api/v2/keyspaces"
+
+// NewKeyspaceCommand return a keyspace subcommand of rootCmd
+func NewKeyspaceCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "keyspace [command] [flags]",
+		Short: "show keyspace information",
+		Run:   showKeyspaceCommandFunc,
+	}
+	return cmd
+}
+
+func showKeyspaceCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+
+	resp, err := doRequest(cmd, fmt.Sprintf("%s/%s?refresh_group_id=", keyspacePrefix, args[0]), http.MethodGet, http.Header{})
```

Comment:
/pd/api/v2/keyspaces/name?refresh_group_id=

Reply:
yes

Reply:
Is it the best practice?

Reply:
No, it is not a best practice, the best practice should be to return the group id of the keyspace meta as the latest, not to add other fields to force it to be refreshed. But it is not suitable for our case.

And renaming `force_refresh_group_id=true` may be much clearer.